{"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/21/comments/43100972.json"}, "html": {"href": "#!/levitsky/pyteomics/issues/21#comment-43100972"}}, "issue": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/21.json"}}, "type": "issue", "id": 21, "repository": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics.json"}, "html": {"href": "#!/levitsky/pyteomics"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3f6302de-10da-4d3c-b258-0bb6a2368972}ts=74456"}}, "type": "repository", "name": "pyteomics", "full_name": "levitsky/pyteomics", "uuid": "{3f6302de-10da-4d3c-b258-0bb6a2368972}"}, "title": "Can't get the mzIdentML schema for version"}, "content": {"raw": "Hello. This is the code that I am trying to implemen. It's really simple:\n\n\n```\n#!python\n\nimport pandas as pd\nfrom pyteomics import mzid\npand = data('example.mzid')\nsequences = pand['PeptideSequence'].tolist()\n\nprint sequences\n```\n\n\nSince I only want the sequences from the file. \nWhen I try the program using the example available in your website works really well, but when I tried to use a different file I got this error: \n\n\n```\n#!python\n\nCan't get the mzIdentML schema for version `1.2.0` from <http://www.psidev.info/files/mzIdentML1.2.0.xsd> at the moment.\nUsing defaults for 1.1.0.\nYou can disable reading the schema by specifying `read_schema=False`.\nTraceback (most recent call last):\n  File \"mz.py\", line 3, in <module>\n    pand = mzid.DataFrame('example2.mzid')\n  File \"/usr/local/lib/python2.7/dist-packages/pyteomics/mzid.py\", line 345, in DataFrame\n    prot_descr.append(d['protein description'])\nKeyError: 'protein description'\n```", "markup": "markdown", "html": "<p>Hello. This is the code that I am trying to implemen. It's really simple:</p>\n<div class=\"codehilite language-python\"><pre><span></span><span class=\"kn\">import</span> <span class=\"nn\">pandas</span> <span class=\"kn\">as</span> <span class=\"nn\">pd</span>\n<span class=\"kn\">from</span> <span class=\"nn\">pyteomics</span> <span class=\"kn\">import</span> <span class=\"n\">mzid</span>\n<span class=\"n\">pand</span> <span class=\"o\">=</span> <span class=\"n\">data</span><span class=\"p\">(</span><span class=\"s1\">&#39;example.mzid&#39;</span><span class=\"p\">)</span>\n<span class=\"n\">sequences</span> <span class=\"o\">=</span> <span class=\"n\">pand</span><span class=\"p\">[</span><span class=\"s1\">&#39;PeptideSequence&#39;</span><span class=\"p\">]</span><span class=\"o\">.</span><span class=\"n\">tolist</span><span class=\"p\">()</span>\n\n<span class=\"k\">print</span> <span class=\"n\">sequences</span>\n</pre></div>\n\n\n<p>Since I only want the sequences from the file. \nWhen I try the program using the example available in your website works really well, but when I tried to use a different file I got this error: </p>\n<div class=\"codehilite language-python\"><pre><span></span><span class=\"n\">Can</span><span class=\"s1\">&#39;t get the mzIdentML schema for version `1.2.0` from &lt;http://www.psidev.info/files/mzIdentML1.2.0.xsd&gt; at the moment.</span>\n<span class=\"n\">Using</span> <span class=\"n\">defaults</span> <span class=\"k\">for</span> <span class=\"mf\">1.1</span><span class=\"o\">.</span><span class=\"mf\">0.</span>\n<span class=\"n\">You</span> <span class=\"n\">can</span> <span class=\"n\">disable</span> <span class=\"n\">reading</span> <span class=\"n\">the</span> <span class=\"n\">schema</span> <span class=\"n\">by</span> <span class=\"n\">specifying</span> <span class=\"sb\">`read_schema=False`</span><span class=\"o\">.</span>\n<span class=\"n\">Traceback</span> <span class=\"p\">(</span><span class=\"n\">most</span> <span class=\"n\">recent</span> <span class=\"n\">call</span> <span class=\"n\">last</span><span class=\"p\">):</span>\n  <span class=\"n\">File</span> <span class=\"s2\">&quot;mz.py&quot;</span><span class=\"p\">,</span> <span class=\"n\">line</span> <span class=\"mi\">3</span><span class=\"p\">,</span> <span class=\"ow\">in</span> <span class=\"o\">&lt;</span><span class=\"n\">module</span><span class=\"o\">&gt;</span>\n    <span class=\"n\">pand</span> <span class=\"o\">=</span> <span class=\"n\">mzid</span><span class=\"o\">.</span><span class=\"n\">DataFrame</span><span class=\"p\">(</span><span class=\"s1\">&#39;example2.mzid&#39;</span><span class=\"p\">)</span>\n  <span class=\"n\">File</span> <span class=\"s2\">&quot;/usr/local/lib/python2.7/dist-packages/pyteomics/mzid.py&quot;</span><span class=\"p\">,</span> <span class=\"n\">line</span> <span class=\"mi\">345</span><span class=\"p\">,</span> <span class=\"ow\">in</span> <span class=\"n\">DataFrame</span>\n    <span class=\"n\">prot_descr</span><span class=\"o\">.</span><span class=\"n\">append</span><span class=\"p\">(</span><span class=\"n\">d</span><span class=\"p\">[</span><span class=\"s1\">&#39;protein description&#39;</span><span class=\"p\">])</span>\n<span class=\"ne\">KeyError</span><span class=\"p\">:</span> <span class=\"s1\">&#39;protein description&#39;</span>\n</pre></div>", "type": "rendered"}, "created_on": "2018-02-05T19:57:48.499429+00:00", "user": {"display_name": "Esteban", "uuid": "{dddf4552-e30d-496e-8482-d2135d035a77}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bdddf4552-e30d-496e-8482-d2135d035a77%7D"}, "html": {"href": "https://bitbucket.org/%7Bdddf4552-e30d-496e-8482-d2135d035a77%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/fae14e2182cfc76ad9d5654b177cc5a3d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsE-5.png"}}, "nickname": "stiven42", "type": "user", "account_id": "5a78b593082eb06819241284"}, "updated_on": null, "type": "issue_comment", "id": 43100972}